### DISCUSSION TITLE
Comment for FLURZnrc spectral output regions is incorrect

### DISCUSSION THREAD

**Discussion by Turkiphys** [nrc-cnrc/EGSnrc#1046] [Questions] [answered]
Hi

I am trying to plot the fluence spectrum (fluence vs energy) that crossing a specific slab using FLURZnrc user code. So I am trying to output the data to a `.spectra` file. However, the `flurznrc_template.egsinp` file says: 

```
START SPECTRAL PLOT IN REGION= 2, 10
 STOP SPECTRAL PLOT IN REGION=  2, 10
                                #start, stop pairs of regions.  For each
                                #pair, a fluence spectrum (ie fluence vs
                                #energy) will be output to the .spectra
                                #file.  In this case, spectra will be
                                #output for the slab between Z=10 cm and Z=12 cm
                                #(see geometry above).  Each radial region
                                #will have its own spectrum
```

And when I look at the geometry, the last slab is at 10 cm (not 12 cm). As I understand from the manual, the regions are numbered from the top to bottom and innermost radius to outermost as shown in the attached screenshot of the geometry. Am I missing something or misunderstood the geometry?  

So if I’d like to evaluate and output the fluence vs energy just below the air region (in the `flurznrc_template.egsinp`, shown in the attached) to a `.spectra` file, should I put:

```
START SPECTRAL PLOT IN REGION= 12
STOP SPECTRAL PLOT IN REGION=  12  ?
```

Any help would be appreciated

Turki 

![Screenshot 2023-08-04 at 04 53 09](https://github.com/nrc-cnrc/EGSnrc/assets/141866218/c291a9c3-8f82-415f-90a0-5cb0fbac5fce)

**Comment by ftessier** [comment, marked as answer]
This is what I understand, indeed: one start and stop region pair, here `12`. Note that fluence is computed over the specified region _volume,_ not at the surface below (or above).

**Reply by ftessier** [reply]
But I don't understand the comment in the manual `the slab between Z=10 cm and Z=12 cm`. I wonder if this is a comment that was not updated when the template was changed? I'll check.

**Reply by ftessier** [reply]
The relevant parts of the `flurznrc_template.egsinp` file [have not been changed](https://github.com/nrc-cnrc/EGSnrc/blame/fe83057e6db99768a281e343498e3eedf185206d/HEN_HOUSE/user_codes/flurznrc/flurznrc_template.egsinp) since at least 2015 ([Line 141](https://github.com/nrc-cnrc/EGSnrc/blame/fe83057e6db99768a281e343498e3eedf185206d/HEN_HOUSE/user_codes/flurznrc/flurznrc_template.egsinp#L141-L142), [Line 484](https://github.com/nrc-cnrc/EGSnrc/blame/fe83057e6db99768a281e343498e3eedf185206d/HEN_HOUSE/user_codes/flurznrc/flurznrc_template.egsinp#L484-L485)).

**Reply by ftessier** [reply]
I looked further in the logs, the lines in question were entered as is when the file was added into the version control at the time (CVS), on Jul 11, 2003.

**Reply by ftessier** [reply]
The `Z=10 cm" and "Z=12 cm" are found in the other templates, so I can only guess that this was copied over from another template, but not adjusted according to the geometry.

**Comment by ftessier** [comment]
I am converting this to issue #1047, so we don't forget to update it.

**Reply by Turkiphys** [reply]
Thanks for your help, appreciated.
